### PR TITLE
Add Spanish Translation

### PR DESCRIPTION
**Pull request by Angelfire**
- [x] index.js
- [x] Translate cheatsheet.json
- [x] Translate general.json
- [x] Translate landing.json
- [x] Translate learn.json

### CODE REVIEW

**Comment by aykutkardas** [comment]
This is great! 🎉 I will review. Thanks. @Angelfire

File: src/localization/es/general.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  "general.text": "Texto",
+  "general.completedStep": "¡Paso Completado!",
+  "general.hintQuestion": "Mostrar respuesta",
+  "general.reportStep": "Reportar un problema",
+  "general.startLearning": "Comienza a aprender",
+  "general.comingSoon": "Próximamente",
+  "general.becomeSponsor": "Convertirse en un patrocinador",
+  "general.learn": "Aprender",
+  "general.playground": "Playground",
+  "general.cheatsheet": "Hoja de referencia",
```
Comment (aykutkardas) [inline src/localization/es/general.json:15]: Actually, `Cheatsheet` is a term. It seems better to use it as is.

File: src/localization/es/general.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  "general.ourSponsors": "Nuestros patrocinadores",
+
+  "notFound.intro": "La página que está buscando no existe.",
+  "notFound.button": "Volver al home",
+
+  "page.landing.title": "Regex Learn - Paso a paso, de cero a avanzado.",
+  "page.learn.title": "Regex Learn - RegEx Curso Interactivo",
+  "page.cheatsheet.title": "Regex Learn - Hoja de referencia RegEx",
```
Comment (aykutkardas) [inline src/localization/es/general.json:23]: [#comment](https://github.com/aykutkardas/regexlearn.com/pull/94/files?authenticity_token=OFFV2CY0i1bME4yVtJH3aO%2BlqsPXKqGbK78fMvWK2e2e4DnjEweHDP%2FX5t0cpMrv5aEWeG%2FqxcsKmBTJa2pkuQ%3D%3D&file-filters%5B%5D=.json#r764289055)

```json
  "page.cheatsheet.title": "Regex Learn - RegEx Cheatsheet",
```

File: src/localization/es/learn.json [hunk 3]
```diff
@@ -0,0 +1,163 @@
+{
+  "examples.starter.title": "Práctica",
+  "examples.starter.description": "Ahora es el momento de hacer práctica algo para consolidar lo que hemos aprendido.",
+
+  "examples.finish.title": "¡Practica completada!",
+  "examples.finish.description": " ",
+
+  "steps.starter.title": "Intro",
+  "steps.starter.description": "RegEx es la abreviación de `Regular Expression`. Ayudar a coincidir, encontrar o gestionar el texto. Comience escribiendo `OK` en el campo Regex para pasar al primer paso y acceder a la descripción más detallada.",
```
Comment (aykutkardas) [inline src/localization/es/learn.json:9]: `Ayudar a coincidir, encontrar o gestionar el texto.` 

It should look like the following. But I am not sure. 🔽 

`Ayuda a hacer coincidir, buscar o administrar texto.`

File: src/localization/es/learn.json [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+  "steps.asterisk.title": "Asterisco `*`",
+  "steps.asterisk.description": "Ponemos un asterisco `*` al inicio para indicar que un carácter no coincide en absoluto o puede coincidir muchas veces. Por ejemplo, indica que la letra `e` nunca debe ocurrir en el texto, o puede ocurrir una o más de lado a lado.",
+
+  "steps.plusCharacter.title": "Signo más `+`",
+  "steps.plusCharacter.description": "Para indicar que un carácter puede ocurrir una o más veces, ponemos el signo más `+` al final. Por ejemplo, indica que la letra `e` puede ocurrir una o más veces en el texto.",
+
+  "steps.questionMark.title": "Signo de interrogación `?`",
+  "steps.questionMark.description": "Para indicar que un carácter es opcional, ponemos un signo de interrogación `?` al final. Por ejemplo, indica que la siguiente letra `u` es opcional.",
+
+  "steps.quantifier.title": "Llaves - 1",
```
Comment (aykutkardas) [inline src/localization/es/learn.json:62]: Isn't `"Llaves"` actually `"Tirantes Rizados"`?    
`{}`

Comment (Angelfire) [reply src/localization/es/learn.json:62]: "_Tirantes Rizados_" is not a thing in spanish, here you can find the names in spanish:
- parenthesis = paréntesis
- curly braces = llaves
- brackets = corchetes

image

**Review by aykutkardas — APPROVED** [review]

**Pull request merged**